example/spec/helpers/Collections.js

```diff
@@ -69,8 +69,16 @@ const removeCollectionAndAllDependencies = async (params) => {
   console.log('Execution Cleanup Complete:');
   console.log(executionsForDeletion);
   console.log(executionDeletionResult);
-
-  await deleteCollection({ prefix, collectionName: collection.name, collectionVersion: collection.version });
+  try {
+    await getCollection({
```

Comment:
Hmm that's a good point! I didn't think of that [`efeafc9` (#2533)](https://github.com/nasa/cumulus/pull/2533/commits/efeafc9ee4d5b539e625a73982600605c508f082)

Reply:
I've since changed this file [changes here](https://github.com/nasa/cumulus/pull/2533/files#diff-170b315807ad89c1b2e7950afc617557e2c7cbdda27bdbd7fa03546a0920f194) 

For some reason, moving the block higher to check that a collection exists before deleting all dependencies lead to other issues (failing cleanup) that I couldn't quite understand. Instead, I updated this to check that each dependency exists before deleting